lib/constants/index.ts

```diff
@@ -302,3 +302,10 @@ export const verra = {
 export const goldStandard = {
   projectDetailPage: `${REGISTRIES.GoldStandard.url}/projects/details`,
 };
+
+/**
+ * Used to create signed messages during authentication flow
+ * See: "https://github.com/KlimaDAO/klimadao/wiki/Carbonmark#authentication",
+ */
+export const CARBONMARK_AUTHENTICATION_MESSAGE =
```

Comment:
don't see this used anywhere did you intend to import and replace `editSignMessage`?

Reply:
This will be in a seperate PR to clean up the env vars in vercel, removed now